### PR TITLE
Add ERC: Distinguishable base256emoji Addresses

### PR DESCRIPTION
**Pull request by wjmelements**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by wjmelements** [comment]
Needs to be assigned ERC number since it's no-longer just the PR. Otherwise this would be ERC-354.

**Comment by wjmelements** [comment]
> Error: file name "erc-draft_distinguishable_account_addresses" not in correct format

hmm I really thought it was in the correct format. Perhaps it only wants the ERC number to be assigned.

From ERC-Template:
> When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`.

I picked the name `ERCS/erc-draft_distinguishable_account_addresses.md` therefore since this is the ERC repo, after a assuming the `eip-` template was a remnant of the repository's split from EIPs. Perhaps it is not sufficiently abbreviated.

File: ERCS/erc-draft_distinguishable_account_addresses.md [hunk 1]
```diff
@@ -0,0 +1,372 @@
+---
```
Comment (abcoathup) [inline ERCS/erc-draft_distinguishable_account_addresses.md:1]:
```suggestion
---
eip: 7673
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.

File: ERCS/erc-draft_distinguishable_account_addresses.md [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+title: Distinguishable Account Addresses 
+description: Depict Account Addresses As A String of 20 Emoji 
+author: William Morriss (@wjmelements)
+discussions-to: https://ethereum-magicians.org/t/erc-distinguishable-account-addresses/19461
```
Comment (abcoathup) [inline ERCS/erc-draft_distinguishable_account_addresses.md:5]:
```suggestion
discussions-to: https://ethereum-magicians.org/t/erc-7673-distinguishable-account-addresses/19461
```

File: ERCS/erc-draft_distinguishable_account_addresses.md [hunk 3]
```diff
@@ -0,0 +1,372 @@
+---
+title: Distinguishable Account Addresses
```
[the remark this answers is not in the document]
Comment (wjmelements) [reply ERCS/erc-draft_distinguishable_account_addresses.md]:
```
3 | title: Distinguishable Account Addresses Using base256emoji
  |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ too long
```

[the remark this answers is not in the document]
Comment (wjmelements) [reply ERCS/erc-draft_distinguishable_account_addresses.md]: I wonder what is the limit

**Comment by wjmelements** [comment]
```
For the Links > Internal check, the following failures were found:

* At ./_site/EIPS/eip-7673.html:208:

  internally linking to ./erc-20.md, which does not exist

* At ./_site/EIPS/eip-7673.html:[15](https://github.com/ethereum/ERCs/actions/runs/8517744456/job/23328806087?pr=354#step:8:16)38:

  internally linking to ./erc-55.md, which does not exist
```
hmm

File: ERCS/erc-7673.md [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+discussions-to: https://ethereum-magicians.org/t/erc-7673-distinguishable-account-addresses/19461
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-04-01
+---
+
+## Abstract
+
+Immediately replace the hexadecimal address encoding with base256emoji in all user interfaces.
```
Comment (abcoathup) [inline ERCS/erc-7673.md:15]: Perhaps more of a recommendation for the primary display and use of addresses?

Comment (wjmelements) [reply ERCS/erc-7673.md:15]: The current display is [not sufficiently idiot-proof](https://ethereum-magicians.org/t/patch-proposals-for-the-ongoing-wallet-homomoroph-attacks/16686/9?u=wjmelements) and is causing losses so ERC-55 should be replaced as soon as possible. The rationale is similar to ERC-55 so I would copy the style of that abstract but that ERC does not have an abstract.

**Comment by SamWilsn** [comment]
I'd just like to confirm that you want to pursue this before I invest any time in reviewing, considering the day it was posted.

**Comment by wjmelements** [comment]
Yes please

File: ERCS/erc-7673.md [hunk 5]
```diff
@@ -0,0 +1,373 @@
...
+discussions-to: https://ethereum-magicians.org/t/erc-7673-distinguishable-account-addresses/19461
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-04-01
+---
+
+## Abstract
+
+Introduce base256emoji for use as the primary input and display for account addresses in all user interfaces.
```
Comment (SamWilsn) [inline ERCS/erc-7673.md:15]: It'd be nice to expand a little bit on why you want this, and how your proposal achieves its goal. The abstract should give a high-level but technical overview to the reader.

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**